editor/src/Components/DataTable.js

```diff
@@ -160,8 +173,10 @@ const DataTable = ({
                                         autoFocus
                                         name={field}
                                         id={field}
-                                        onChange={e => handleInputChange(e)}
-                                        defaultValue={fieldValue}
```

Comment:
im updating all of these inputs to be [controlled components](https://blog.logrocket.com/controlled-vs-uncontrolled-components-in-react/) instead of uncontrolled